third_party/open-policy-agent/gatekeeper/helmify/static/values.yaml

```diff
@@ -1,7 +1,86 @@
-controllerManager:
+runtimeConfig:
+  health: {}
+    # healthProbeBindAddress: :8081
+  metrics: {}
+    # bindAddress: 127.0.0.1:8080
+  webhook: {}
+    # port: 9443
+  leaderElection: {}
+    # leaderElect: true
+    # resourceName: e29e094a.k8s.io
+  manager:
+    runtime: containerd
+    otlpEndpoint: ""
+    logLevel: info
+    scheduling: {}
+      # repeatInterval: "" # to be parsed into time.Duration
```

Comment:
`# to be parsed into time.Duration` this is more implementation detail. can we remove or describe?

Reply:
I'll remove. In the documentation we'll have to make sure it's understood that a unit needs to be specified, ex. 23h

Reply:
done